types/heremaps/index.d.ts

```diff
@@ -5681,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes, opt_locale?: H.ui.i18n.Localization | string): UI;
+            static createDefault(map: H.Map, mapTypes: H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
Is the reason for this change that the `MapTypes` indexer requires its members be assignable to `MapType`, but `DefaultLayers` has members that are `Layer` subtypes? Without knowing the internals of the heremaps JavaScript API, I couldn't say whether passing in a `DefaultLayers` to this would be a problem. 

Alternatively, `H.service.Platform.MapTypes` could have its indexer widened to: `H.service.MapType | H.map.layer.Layer`.